torch/csrc/jit/init.cpp

```diff
@@ -563,6 +566,16 @@ void initJITBindings(PyObject* module) {
   tracer::initPythonTracerBindings(module);
   script::initTreeViewBindings(module);
   script::initJitScriptBindings(module);
+
+  setPrintHandler([](const std::string& str) {
+    AutoGIL ag;
+    // TODO: I can't get the below to work without crashing.
+    //       figure out how to make this work. Otherwise,
+    //       WriteStdout will truncate to 1000 bytes.
+    // auto _stdout = py::module::import("sys").attr("stdout");
+    // _stdout.attr("write")(_stdout, str);
+    PySys_WriteStdout("%s", str.c_str());
```

Comment:
can we not just call print through pybind?

Reply:
I can't get it to not complain that it expects bytes, not str, even if i explicitly cast to py::bytes. something seems broken there in pybind